Unity/Assets/MyScripts/MyAppStart.cs

```diff
@@ -29,67 +36,84 @@ void Start ()
 
 		Debug.Log ("unity-script: unity version" + IronSource.unityVersion ());
 
-		// Add Banner Events
-		IronSourceBannerEvents.onAdLoadedEvent += BannerOnAdLoadedEvent;
-		IronSourceBannerEvents.onAdLoadFailedEvent += BannerOnAdLoadFailedEvent;
-		IronSourceBannerEvents.onAdClickedEvent += BannerOnAdClickedEvent;
-		IronSourceBannerEvents.onAdScreenPresentedEvent += BannerOnAdScreenPresentedEvent;
-		IronSourceBannerEvents.onAdScreenDismissedEvent += BannerOnAdScreenDismissedEvent;
-		IronSourceBannerEvents.onAdLeftApplicationEvent += BannerOnAdLeftApplicationEvent;
-
 		// SDK init
 		Debug.Log ("unity-script: IronSource.Agent.init");
-		IronSource.Agent.init (appKey);
-		//IronSource.Agent.init (appKey, IronSourceAdUnits.REWARDED_VIDEO, IronSourceAdUnits.INTERSTITIAL, IronSourceAdUnits.OFFERWALL, IronSourceAdUnits.BANNER);
-        //IronSource.Agent.initISDemandOnly (appKey, IronSourceAdUnits.REWARDED_VIDEO, IronSourceAdUnits.INTERSTITIAL);
-
-        //Set User ID For Server To Server Integration
-        //// IronSource.Agent.setUserId ("UserId");
 		
-		// Load Banner example
-		IronSource.Agent.loadBanner (IronSourceBannerSize.BANNER, IronSourceBannerPosition.BOTTOM);
+		LevelPlay.Init(appKey,uniqueUserId,new []{LevelPlayAdFormat.REWARDED});
+		
+		LevelPlay.OnInitSuccess += OnInitializationCompleted;
+		LevelPlay.OnInitFailed += (error => Debug.Log("Initialization error: " + error));
 	}
 	
-	// Update is called once per frame
-	void Update ()
+	void LoadBanner ()
 	{
+		// Create object
+		bannerAd = new LevelPlayBannerAd(bannerAdUnitId);
+		
+		bannerAd.OnAdLoaded += BannerOnAdLoadedEvent;
+		bannerAd.OnAdLoadFailed += BannerOnAdLoadFailedEvent;
+		bannerAd.OnAdDisplayed += BannerOnAdDisplayedEvent;
+		bannerAd.OnAdDisplayFailed += BannerOnAdDisplayFailedEvent;
+		bannerAd.OnAdClicked += BannerOnAdClickedEvent;
+		bannerAd.OnAdCollapsed += BannerOnAdCollapsedEvent;
+		bannerAd.OnAdLeftApplication += BannerOnAdLeftApplicationEvent;
+		bannerAd.OnAdExpanded += BannerOnAdExpandedEvent;
+		
+		// Ad load
+		bannerAd.LoadAd();
 
 	}
 
-	void OnApplicationPause (bool isPaused)
+	void OnInitializationCompleted(LevelPlayConfiguration configuration)
 	{
-		Debug.Log ("unity-script: OnApplicationPause = " + isPaused);
-		IronSource.Agent.onApplicationPause (isPaused);
+		Debug.Log("Initialization completed");
+		LoadBanner();
+
 	}
 
 	//Banner Events
-	void BannerOnAdLoadedEvent(IronSourceAdInfo adInfo)
+	void BannerOnAdLoadedEvent(LevelPlayAdInfo adInfo)
 	{
 		Debug.Log("unity-script: I got BannerOnAdLoadedEvent With AdInfo " + adInfo);
```

Comment:
adInfo.ToString()?

Reply:
we don't need `ToString` because we override it already